examples/footer/index.html

```diff
@@ -8,59 +8,85 @@
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/u-s-privacy-policy-statement.html" title="U.S. Privacy Policy Statement">U.S. Privacy Policy Statement</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal.html" title="Disclaimer / Registrations / Copyright Statement">Disclaimer / Registrations / Copyright Statement</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/linking-policy.html" title="Linking Policy">Linking Policy</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-privacy-practices.html" title="Notice of Privacy Practices (HIPAA)">Notice of Privacy Practices (HIPAA)</a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.stjude.org/legal/notice-of-non-discrimination.html" title="Notice of Non-Discrimination">Notice of Non-Discrimination</a>
-          </li>
-        </ul>
-        <ul class="social-media">
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.facebook.com/stjuderesearch" class="fab fa-facebook-f">
-              <span class="sr-only">Facebook</span>
+    <footer>
+      <div class="container">
+        <div class="row">
+          <div class="col-6 col-lg-3 col-md-4 offset-xl-0 offset-lg-5 offset-md-4 offset-md-3 offset-3 footer-logo">
+            <a class="navbar-brand" href="https://stjude.cloud">
+              <span class="logo"></span>
             </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://twitter.com/StJudeResearch" class="fab fa-twitter">
-              <span class="sr-only">Twitter</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.instagram.com/stjuderesearch/" class="fab fa-instagram">
-              <span class="sr-only">Instagram</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/company/st-jude-childrens-research-hospital/" class="fab fa-linkedin">
-              <span class="sr-only">LinkedIn</span>
-            </a>
-          </li>
-          <li>
-            <a target="_blank" rel="noopener noreferrer" href="https://www.youtube.com/user/MyStJude" class="fab fa-youtube">
-              <span class="sr-only">YouTube</span>
-            </a>
-          </li>
-        </ul>
+            <a class="sjc-title" href="https://stjude.cloud">St. Jude Cloud</a>
+          </div>
+          <div class="col-md-3 col-6 offset-xl-0 offset-md-2 footer-about">
+            <h6>About St. Jude</h6>
+            <ul>
+              <li>
+                <a href="https://www.stjude.org">Home</a>
```

Comment:
Call me a dumb user but "Home" in the footer (even under the sub-heading of "About St. Jude" albeit next to our SJC logo), I still missed that the link would be going to the hospital's homepage initially...

I think what I'm missing is the context that SJC is powered by the hospital (which has been an on-going conversation at the top-level too), and I fear that the "About St. Jude" header isn't strong enough. Is there an opportunity to make that connection in the footer before we add several links to the hospital's website?

Alternatively, how many links back to the hospital's website do we need, @claymcleod and @kirbyjbirch? [Together's ](https://together.stjude.org/en-us/)footer links are all internal.

Reply:
I think it's the fact that we have the cloud logo featured prominently before users are naturally arriving at those links. I too would expect the footer links to be cloud oriented. Even the social links are pointing to non-cloud specific accounts.